extensions/amp-date-countdown/1.0/amp-date-countdown.js

```diff
@@ -120,6 +120,10 @@ AmpDateCountdown['props'] = {
   'whenEnded': {attr: 'when-ended', type: 'string'},
   'locale': {attr: 'locale', type: 'string'},
   'biggestUnit': {attr: 'biggest-unit', type: 'string'},
+  'countUp': {
+    attrs: ['data-count-up'],
+    parseAttrs: (el) => el.hasAttribute('data-count-up'),
+  },
```

Comment:
This is to match the behavior in `0.1` which checks if the attribute `data-count-up` exists at all.  So `data-count-up="false"` should still register as the component SHOULD do the count up functionality.

Doing `attr: 'data-count-up', type: 'boolean'` does not catch this use case properly.

Can also consider updating `0.1` if this behavior seems to strange.

Reply:
Let's just do `boolean` and we will break backward compat here.

Reply:
Okay sounds good, updated!